src/main/java/net/shadowmage/ancientwarfare/npc/entity/faction/NpcFaction.java

```diff
@@ -172,6 +174,16 @@ public EnumCreatureAttribute getCreatureAttribute() {
 			return EnumCreatureAttribute.UNDEFINED;
 	}
 
+	@Override
+	public boolean isPotionApplicable(PotionEffect potioneffectIn) { // makes lizardman and coven immune to poison
+		if (potioneffectIn.getPotion() == MobEffects.POISON && getFaction().equals("lizardman|coven")) {
+			net.minecraftforge.event.entity.living.PotionEvent.PotionApplicableEvent event = new net.minecraftforge.event.entity.living.PotionEvent.PotionApplicableEvent(this, potioneffectIn);

```

Comment:
the full package name for the class is unnecessary, should really go into import so that only the class name is used here, the same for the EVENT_BUS call below.